<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>116</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Galileo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="altitude_label">
        <property name="text">
         <string>Altitude: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="elevation_label">
        <property name="text">
         <string>Elevation: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="elevation">
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="maximum">
         <double>1.560000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="km_mi">
        <item>
         <property name="text">
          <string>km</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>miles</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="rad_deg">
        <item>
         <property name="text">
          <string>radians</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>degrees</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="altitude">
        <property name="minimum">
         <double>100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>256</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionReport"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionReport">
   <property name="text">
    <string>Report a bug</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
